src/NavData.tsx

```diff
@@ -100,6 +100,39 @@ const rootNavData: NavMenu = deepFreeze([
           },
         ],
       },
+      {
+        href: '/how-to',
+        title: 'How To',
+        sections: [
+          {
+            href: '/how-to/set-up',
+            title: 'Set Up',
+            sections: [
+              {
+                title: 'Plural CLI',
+                href: '/getting-started/quickstart',
+              },
+              {
+                title: 'Management Cluster',
+                // href: '/how-to/set-up/mgmt-cluster',
+                href: '/deployments/cli-quickstart',
+              },
+              {
+                title: 'Workload Cluster',
+                href: '/how-to/set-up/workload-cluster',
+              },
+              {
+                title: 'SCM Connection',
```

Comment:
you'll need to set up the scm connection before the workload cluster since it's a prereq.  Also many users aren't going to understand the acronym, this should be titled something like `Source Control Integration` or something like that.